requirements.txt

```diff
@@ -1,45 +1,25 @@
-alembic==0.9.6
 aniso8601==1.3.0
-appnope==0.1.0
```

Comment:
errr what happened here? seems out of date? it's ok no one is perfect, but we strive for it!